src/mudlet-lua/tests/DB_spec.lua

```diff
@@ -105,7 +105,7 @@ describe("Tests DB.lua functions", function()
         local results = db:fetch(mydb.enemies)
         assert.is_true(#results == 1)
       end)
-
+			
```

Comment:
empty spaces

Reply:
Trailing spaces in DB_spec.lua trimmed now.